message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -139,6 +139,17 @@
                 <param index="7">Empty</param>
             </entry>
 
+            <entry name="MAV_CMD_DO_ARM_CHECK" value="42429">
+                <description>Reply with whether or not you can arm and why</description>
```

Comment:
How is "and why" done?

Reply:
The way the pre-arm checks are run right now is better than it was in 2016.
And there are just too many bits, we would need a huge bitmask, and we would have to constantly,
change Mavlink when a new check gets added.

I vote that this one should be closed now.